website/content/docs/platform/k8s/injector/index.mdx

```diff
@@ -20,6 +20,19 @@ the request. This functionality is provided by the [vault-k8s](https://github.co
 project and can be automatically installed and configured using the
 [Vault Helm](https://github.com/hashicorp/vault-helm) chart.
 
+## Supported Kubernetes Versions
+
+Each Vault Agent Injector release supports running on a rolling set of the most
+recent [Kubernetes minor releases][k8s-releases]. The following table documents
+the versions tested. Each version of the injector may work with other versions
+of Kubernetes, but those are not supported.
```

Comment:
Should be mention some end-of-support policy here? When does a k8s version pop off the stack of supported versions?

Reply:
Good question. I couldn't really think of a way to make the answer clear with the current wording and table (without also having to support a really long tail of old versions), so I've reworked it to simply be a list of supported Kubernetes versions.